src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,23 @@
+import { constants } from "../utils/ApiConstants";
+import { getKeyFromSession, startStubServiceAndReturnSessionIdByType } from "../utils/ApiTestSteps";
+
+
+describe("/session Happy Path", () => {
+	it("BAV | NO_PHOTO_ID test", async () => {
+		const sessionResponse = await startStubServiceAndReturnSessionIdByType("FACE_TO_FACE");
+		expect(sessionResponse.status).toBe(200);
+		console.log(sessionResponse.data);
```

Comment:
I've noticed the tests are really noisy which can make debugging difficult. Could we please either add some more context to the logs (something like `NO_PHOTO_ID session response ${sessionResponse.data}` or just remove some of them? Not sure all of them are providing value